example/disk-agent.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+if [[ -z $LOCATION_FILE ]]; then
```

Comment:
As is, this check will fail when script is called from `4-start_nbs_storage.sh` because we no longer pass location as variable...

Reply:
This script is not called from ```4-start_nbs_storage.sh```, it's on its own. This PR was intented to introduce ```disk-agent.sh``` wrapper script